tests/spec/examples.py

```diff
@@ -403,7 +403,7 @@ def it_displays_correctly_when_the_service_is_up_json(self, in_example_dir):
                 'state': 'ready',
             },
         }
-        assert stderr == b''
+        assert stderr == ''
```

Comment:
we should probably change the api to return byte for consistency with what people expect (from subprocess)